data/formats-data.js

```diff
@@ -5673,7 +5680,7 @@ exports.BattleFormatsData = {
 		tier: "BL3",
 	},
 	skrelp: {
-		randomBattleMoves: ["scald", "sludgebomb", "thunderbolt", "shadowball", "toxicspikes", "hydropump"],
+		randomBattleMoves: ["scald", "sludgewave", "thunderbolt", "gunkshot", "toxicspikes", "hydropump"],
```

Comment:
because of how damage rolls work in lc, sludge bomb > sludge wave.  The usually deal the same damage, but sludge bomb has the extra poison chance

gunk shot is bad. replace it with hp fire

Reply:
good catch
in my ideal skrelp set gunk shot is the second best move. super powerful, gives mixed threat, dont have to worry about maximizing util of ev spread, hell to the yes. hp fire just seems so weak when adaptability exists

Reply:
hp fire hits ferroseed harder, but that's about it i suppose.  Coverage is still better than putting tspikes on an adaptability mon that just hits hard

Reply:
I can just kill tspikes which seems fine